courses/css/main.css.twig

```diff
@@ -0,0 +1,5 @@
+#course-booking-form form label[for="quantity"],
```

Comment:
Classes here are still pretty specific, the problem here is when a designer goes to work with these they end up having to use similar to higher specificity to beat this. Removing the `form` and `button` parts will make this nicer to work with i think :)